src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapperTest.java

```diff
@@ -129,16 +158,45 @@ public void shouldMapAddressFromCourtFinder() {
             .judicial(Judicial.builder().firstName("Judge").lastName("McJudge").build())
             .referenceNumber("ref no")
             .hearingCourtName("Birmingham Court")
-            .hearingCourtAddress("line1\nline2\nSW1P4BB\nBirmingham")
+            .hearingCourtAddress(CCDAddress.builder()
+                .addressLine1("line1")
+                .addressLine2("line2")
+                .postTown("Birmingham")
+                .postCode("SW1P4BB")
+                .build())
             .docUploadForParty(CCDDirectionPartyType.CLAIMANT)
             .eyewitnessUploadForParty(CCDDirectionPartyType.DEFENDANT)
             .estimatedHearingDuration(CCDHearingDurationType.FOUR_HOURS)
-            .otherDirectionList(ImmutableList.of(
-                OtherDirection.builder()
+            .otherDirections(ImmutableList.of(
+                CCDOrderDirection.builder()
                     .sendBy(LocalDate.parse("2020-10-11"))
                     .directionComment("a direction")
                     .extraOrderDirection(CCDOrderDirectionType.OTHER)
                     .forParty(CCDDirectionPartyType.BOTH)
+                    .build(),
+                CCDOrderDirection.builder()
```

Comment:
Suggestion. I see CCDOrderDirection is used in many tests. is it worth moving these in the Sample Data package as Providers. Thanks

Reply:
👍